### PR TITLE
 Added provision_ignore_aready_exists configuration key to existent e…

### PR DESCRIPTION
**Pull request by javierag**
…lements on provision

### CODE REVIEW

File: main/openchange/ChangeLog [hunk 1]
```diff
@@ -1,4 +1,6 @@
 HEAD
+	+ Added provision_ignore_aready_exists configuration key to skip
```
Comment (sixstone-qq) [inline main/openchange/ChangeLog]: Typo: `already_exists`

File: main/openchange/conf/openchange.conf [hunk 2]
```diff
@@ -15,3 +15,13 @@
 # The amount of instances of SOGo that will be spawned
 # to handle multiple requests simultaneously
 #sogod_prefork=3
+
+# If this key is put to yes, then when provisioning openchange
```
Comment (sixstone-qq) [inline main/openchange/conf/openchange.conf]: s/put/set/

File: main/openchange/conf/openchange.conf [hunk 3]
```diff
@@ -15,3 +15,13 @@
...
+# all already existent directory elements will be ignored
+# instead of raising a error.
```
Comment (sixstone-qq) [inline main/openchange/conf/openchange.conf]: Typo: `an error`

File: main/openchange/conf/openchange.conf [hunk 4]
```diff
@@ -15,3 +15,13 @@
...
+# This applies both to schema elements and regular elelments.
```
Comment (sixstone-qq) [inline main/openchange/conf/openchange.conf]: Typo: `applies to both schema and regular elements`

File: main/openchange/conf/openchange.conf [hunk 5]
```diff
@@ -15,3 +15,13 @@
...
+# Remember that if the ignored elements have incompatible values
+# this will result in a unworking installation.
```
Comment (sixstone-qq) [inline main/openchange/conf/openchange.conf]: typo: `non working installation`

**Comment by sixstone-qq** [comment]
Apart from commit message subject is too long and the typos, the PR looks good to me.

**Comment by javierag** [comment]
I have committed the suggested changes

**Comment by jacalvo** [comment]
What is the use case for this?

**Comment by javierag** [comment]
Errors that let you with a directory half-provisioned

**Pull request merged**